### ISSUE TITLE
Exception on file watcher creation in VS Code with Node 14

### ISSUE THREAD

**Issue by qzmfranklin** [microsoft/pylance-release#1392]
Same as this:

https://github.com/microsoft/vscode-python/issues/16380

**Comment by jakebailey** [comment]
Copying the logs:

```
[Info  - 10:51:53 PM] Pylance language server 2021.6.0 (pyright 209c8040) starting
[Info  - 10:51:53 PM] Server root directory: /home/quzhongming/.vscode-server/extensions/ms-python.vscode-pylance-2021.6.0/dist
[Error - 10:51:54 PM] Exception received when installing recursive file system watcher: TypeError [ERR_FEATURE_UNAVAILABLE_ON_PLATFORM]: The feature watch recursively is unavailable on the current platform, which is being used to run Node.js
[Error - 10:51:54 PM] Exception received when installing recursive file system watcher: TypeError [ERR_FEATURE_UNAVAILABLE_ON_PLATFORM]: The feature watch recursively is unavailable on the current platform, which is being used to run Node.js
[Error - 10:51:54 PM] Exception received when installing recursive file system watcher: TypeError [ERR_FEATURE_UNAVAILABLE_ON_PLATFORM]: The feature watch recursively is unavailable on the current platform, which is being used to run Node.js
[Error - 10:51:54 PM] Exception received when installing recursive file system watcher: TypeError [ERR_FEATURE_UNAVAILABLE_ON_PLATFORM]: The feature watch recursively is unavailable on the current platform, which is being used to run Node.js
[Error - 10:51:54 PM] Exception received when installing recursive file system watcher: TypeError [ERR_FEATURE_UNAVAILABLE_ON_PLATFORM]: The feature watch recursively is unavailable on the current platform, which is being used to run Node.js
[Error - 10:51:54 PM] Exception received when installing recursive file system watcher: TypeError [ERR_FEATURE_UNAVAILABLE_ON_PLATFORM]: The feature watch recursively is unavailable on the current platform, which is being used to run Node.js
```

This file watcher is broken anyway (#923), so this doesn't actually break things at the moment. The most recent version of Pylance is just better about reporting _why_ the watcher failed to be registered.

**Comment by qzmfranklin** [comment]
> This file watcher is broken anyway (#923), so this doesn't actually break things at the moment. The most recent version of Pylance is just better about reporting why the watcher failed to be registered.

Well, I'd like it to be working. Not having pylance is killing.

**Comment by qzmfranklin** [comment]
What can I do to `solve` / `workaround` this error?

Just verified that the `node --version` is`v14.16.0`.

If I manually downgrade it to node12, it worked fine.

**Comment by fukaf** [comment]
So this error won't cause Pylance launching failure? Then why my Pylance just doesn't work.

**Comment by jakebailey** [comment]
This definitely should not cause Pylance to my work. If it's not working, these messages are unrelated and a new issue with the template filled out would be appreciated.

**Comment by jakebailey** [comment]
The next release changes this message to a warning; this will at least mean it's not shown in the default log level so people think something's gone wrong.

File watching outside the workspace is more generally covered in #923 and has yet to be fixed.

**Comment by jakebailey** [comment]
This issue has been fixed in version 2021.6.1, which we've just released. You can find the changelog here: https://github.com/microsoft/pylance-release/blob/main/CHANGELOG.md#202161-9-june-2021

**Comment by Ezra** [comment]
I'm seeing this behavior on v2021.8.0.
```
[Info  - 11:16:30 AM] Pylance language server 2021.8.0 (pyright dfb796d8) starting
[Info  - 11:16:30 AM] Server root directory: /home/ezra_bradford/.vscode-server/extensions/ms-python.vscode-pylance-2021.8.0/dist
Exception received when installing recursive file system watcher: TypeError [ERR_FEATURE_UNAVAILABLE_ON_PLATFORM]: The feature watch recursively is unavailable on the current platform, which is being used to run Node.js
Exception received when installing recursive file system watcher: TypeError [ERR_FEATURE_UNAVAILABLE_ON_PLATFORM]: The feature watch recursively is unavailable on the current platform, which is being used to run Node.js
Exception received when installing recursive file system watcher: TypeError [ERR_FEATURE_UNAVAILABLE_ON_PLATFORM]: The feature watch recursively is unavailable on the current platform, which is being used to run Node.js
```

VS Code 1.59.0
Windows 10 Home 20H2; WSL-remote-connected to Debian GNU/Linux 11 (bullseye)
Python extension v2021.8.1105858891
Pylance extension v2021.8.0

**Comment by jakebailey** [comment]
Reopening; a recent refactor to this code has changed how we log it so that it's present again.

Again, I will assert that this does not affect anything negatively; we know that these specific watchers fail, but they do not actually cause any problems for the analysis except that #923 is not yet fixed.

**Comment by jakebailey** [comment]
I've re-fixed this for the next release.

**Comment by heejaechang** [comment]
fixed in 2021.8.1 release.

**Comment by CarlosDomingues** [comment]
I'm seeing those errors in my logs when using VS Code 1.63.2 + Pylance 2022.1.1 (pyright 4f61d443)
